specs/Checkbox.md

```diff
@@ -156,9 +129,10 @@ Removing the following two props because the ARIA spec dictates role='checkbox'
 | `ariaLabelledBy`             | User provided        | &#x274C;               | &#x274C;         | &#x274C;              |
 | `ariaPositionInSet`          | Won't be transitioned| &#x274C;               | &#x274C;         | &#x274C;              |
 | `ariaSetSize`                | Won't be transitioned| &#x274C;               | &#x274C;         | &#x274C;              |
```

Comment:
I believe all aria* attributes we weren't going to carry over the camelCased variants and instead use aria-* native attributes.

Reply:
The most recent thing I remembered (from before the holidays) was the opposite, using camelCased instead of native.

Reply:
as the acccessibility behaviors are platform independent, we opted for using aria-* initially istead of camel case. camel case is react specific and having just one approach in both components and behaviors brings consistency. @ecraig12345 it would be good to cover these decisions in design principles, so that we can refer to them and discsus them - I see pros and cons of this but the dicsussion might be lost when done under checkbox.